releasenotes/README.md

```diff
@@ -12,9 +12,41 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v50 release notes.](./v50.md)
 
-## April 8, 2021
+## 51.7.1 (April 8, 2021) - CLI 7.95.0
```

Comment:
this RC will be `51.8.0`.  `51.7.1` was that little extra-rc we did after `51.7.0` to get the rest/soap reversion in